templates/lobby.html

```diff
@@ -508,6 +515,21 @@
             applyRounds(parsed.data.round, maxRounds);
             applyPlayers(parsed.data.players);
 
+            drawer = "";
+            cached = parsed.data.players;
+            cached.forEach(function (player) {
```

Comment:
I'd make this a traditional for-loop instead and add the logic that opens the dialog as soon as you got a match and then add a break to prevent useless iterations.

e.g.

```
for(...) {
   if( match ...) {
        show-dialog-and-stuff
        break
   }
}
```

This way, the codeflow is more clear.